GoRest.Api.Client/Client/Builders/Users/PatchUpdateUserBuilder.cs

```diff
@@ -0,0 +1,44 @@
+﻿using GoRest.Api.Client.Client.Models;
+
+namespace GoRest.Api.Client.Client.Builder
+{
+    public class PatchUpdateUserBuilder
+    {
+        private readonly UpdateUserModel _createUser;
+        public PatchUpdateUserBuilder()
+        {
+            this._createUser = GetDefault();
+        } 
+
+        public PatchUpdateUserBuilder With(Action<UpdateUserModel> updateAction)
+        {
+            updateAction.Invoke(this._createUser);
+            return this;
+        }
+
+        public UpdateUserModel Build()
+        {
+            return this._createUser;
+        }
+
+        public UpdateUserModel Build(Gender gender, Status status)
```

Comment:
not the best approach here. lets discuss offline

Reply:
Done